File: Bussiness Logic (Tier II)/ManagingUsers/Models/Domain/WishList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ManagingUsers.Models.Domain
{
    public class WishList
    {
        public List<Wish> wishList { get; set; }

        public WishList()
        {
            this.wishList = new List<Wish>();
        }


        public void registerWish(string Title, string URL_)
        {
            Wish wish = new Wish(Title, URL_);
            wishList.Add(wish);
        }

        public void removeWish(Wish wish)
        {
            foreach (var wishes in wishList)
            {
                if (wish.Equals(wishes))
                {
                    wishList.Remove(wish);
                }
            }

        }
    }
}

File: Database_Java_V7/src/domain/WishList.java
package domain;

import java.util.ArrayList;

public class WishList {
    private ArrayList<Wish> wishList;

    public WishList() {
        this.wishList = new ArrayList<Wish>();
    }

    public void addWish(String url) {

        Wish wish = new Wish(url);
        wishList.add(wish);
    }

    public void removeWish(Wish wish) {


        if (wishList.contains(wish)) {
            wishList.remove(wish);
        }
    }
}
